README.md

```diff
@@ -174,12 +174,15 @@ For more details, see the [update system documentation](sources/updater/).
 
 ### Update methods
 
+#### Automated updates
+
 There are several ways of updating your Bottlerocket hosts.
 
 For EKS variants of Bottlerocket, we recommend using the [Bottlerocket update operator](https://github.com/bottlerocket-os/bottlerocket-update-operator) for automated updates.
-You can also use one of the methods below for direct control of updates.
 
-For the ECS preview variant of Bottlerocket, we recommend updating hosts using one of the methods below, until further automation is ready.
+For the ECS variant of Bottlerocket, we recommend using the [Bottlerocket ECS updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater/) for automated updates.
+
+For any variant, you can also use the update API for direct control of updates.
```

Comment:
```suggestion
```
With the new "automated updates" header, this doesn't seem like it belongs.  The "update API" section immediately follows, and isn't variant-specific.  See also my other suggestion about putting some of this wording above.

Reply:
Done